wallet/build.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
     compile 'com.squareup.moshi:moshi:1.6.0'
     compile 'org.slf4j:slf4j-api:1.7.28'
     compile 'com.github.tony19:logback-android:2.0.0'
+    compile 'com.android.support:appcompat-v7:23.2.0'
```

Comment:
We don't use AppCompat currently. The App requires Android 5 anyway which includes most of AppCompat.